vue/src/components/HelloWorld.vue

```diff
@@ -2,30 +2,86 @@
   <div class="hello">
     <h1>{{ msg }}</h1>
     <p>
-      For a guide and recipes on how to configure / customize this project,<br>
+      For a guide and recipes on how to configure / customize this project,<br />
       check out the
-      <a href="https://cli.vuejs.org" target="_blank" rel="noopener">vue-cli documentation</a>.
+      <a href="https://cli.vuejs.org" target="_blank" rel="noopener"
+        >vue-cli documentation</a
+      >.
     </p>
     <h3>Installed CLI Plugins</h3>
     <ul>
-      <li><a href="https://github.com/vuejs/vue-cli/tree/dev/packages/%40vue/cli-plugin-babel" target="_blank" rel="noopener">babel</a></li>
-      <li><a href="https://github.com/vuejs/vue-cli/tree/dev/packages/%40vue/cli-plugin-eslint" target="_blank" rel="noopener">eslint</a></li>
+      <li>
+        <a
+          href="https://github.com/vuejs/vue-cli/tree/dev/packages/%40vue/cli-plugin-babel"
+          target="_blank"
+          rel="noopener"
+          >babel</a
+        >
```

Comment:
This kind of HTML formatting by Prettier is something I really don't like, even though I understand some of the motivation (white space after the `>` is significant in HTML and I believe in Vue templates as well).

This is why I've excluded `.html` files from Prettier formatting so far in this repo. We don't exclude HTML files from Prettier formatting in the StackBlitz classic editor currently, but we might consider it because it gets reported as a bug every so often. :/

For most Vue files (which are getting formatted in this PR), having Prettier formatting is useful because it will nicely format JS and CSS code, and JS code inside template attributes. When you don't have too many HTML elements in the template, the results are also nice enough. But when you have a bunch of static HTML content in the template like in this file, then we're back with the strange HTML formatting.

Prettier doesn't have an option for preventing formatting of HTML in Vue/Svelte components. The only option that could apply here is this one, but I'm afraid of its effects on JSX for React projects:
https://prettier.io/docs/en/options.html#bracket-line

All in all, I don't love this output, but the pros still outweigh the cons IMHO.

Reply:
I agree this is a pretty gnarly result we can sometimes get with the templates, but I'm concerned that we might be throwing a baby out with the bathwater by turning the formatting off entirely. 

Currently, some users might find the formatting output weird, others will recognize that it is "prettier-weird" and in that sense, correct. After the change, won't it look like the HTML (/svelte/..) formatting is even more broken – to everyone – as nothing gets formatted?

Reply:
Hmm I don’t know. To clarify, what would your suggestion be?
Is there something you would do differently in this PR?

In the StackBlitz classic editor we currently format `.vue` files with Prettier on save, so pre-formatting the `.vue` files in starter templates using the same settings makes sense (even if I don't love the result).

We currently don't support formatting `.svelte` files, so we could keep `.svelte` files alone in this PR, or format them at least so that the JS and CSS in them uses the same coding conventions as standalone `.js` and `.css` files (the current option).

Reply:
I think I'm fine with keeping the templates as they were then. How about pure HTML files? They are currently being formatted, no?

Reply:
This is what is being formatted in the classic editor and in this PR when it comes to HTML-like content:

| File extension | Formatted in this PR | Formatted on change in the classic editor |
| `.html` | No (ignored) | Yes |
| `.md` | No (ignored) | Yes |
| `.svelte` | Yes | No (not supported) |
| `.vue` | Yes | Yes |

If we want to align this PR with what the classic editor does, we should format `.html`, `.md` and `.vue`, but not `.svelte`.

Another option is to _not_ format any of these HTML-adjacent file types in this PR, which is different from what the classic editor does. This might be a good approach if we plan to revisit what our Prettier integration does, following some user complaints about the formatting creating issues in Markdown and some templating languages. (The main complaint, from the top of my mind, is for files that use a DSL on top of Markdown but use the `.md` extension.)

For now I'll roll back the formatting of `.vue` and `.svelte` files.